### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lakme22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -446,3 +446,7 @@ Open source means that someone from Australia can make a Pull Request to help me
 
 Open source to me is all about the community and the discussions which are a means for collective learning. It is a better learning medium than even a classroom because you can talk and learn from experts in their own fields.
 Open source projects are a living, breathing entity constantly being shaped and molded by each contributor as they addd a little bit of themselves into the project.
+
+###[Mareeswari Lakshmi](https://lakme22.github.io)
+
+Open source is a place where we can learn about team work. Its all about learning together in a community. Finally I have learnt how to contribute to open source , a place from where I learnt so many things. It encourages me to share my knowledge and also to learn from others.
```
Comment (justinclift) [inline README.md]: Small typo here:

&nbsp; &nbsp; "open source , a place"

Should instead be:

&nbsp; &nbsp; "open source, a place"

Commas (,) never have a space right before them.  It's always `some words, then some more words`. :smile:

Are you ok to update this? :smile:

Comment (lakme22) [reply README.md]: Thanks, I will fix it.

File: README.md [hunk 2]
```diff
@@ -449,4 +449,4 @@ Open source projects are a living, breathing entity constantly being shaped and
 
 ###[Mareeswari Lakshmi](https://lakme22.github.io)
 
-Open source is a place where we can learn about team work. Its all about learning together in a community. Finally I have learnt how to contribute to open source , a place from where I learnt so many things. It encourages me to share my knowledge and also to learn from others.
+An open source is a place where we can learn about teamwork. Its all about learning together in a community. Finally, I have learnt how to contribute to open source, a place from where I learnt so many things. It encourages me to share my knowledge and also to learn from others.
```
Comment (justinclift) [inline README.md]: "team work" is definitely two words. :smile:

Comment (lakme22) [reply README.md]: Thanks, I will fix it.

**Comment by justinclift** [comment]
Yep, the text of it looks good now. :smile:

GitHub is giving an error though, as it's wanting your entry to be added after @thirulak's.

It might be a bit complicated to fix, but I can do it manually pretty easily.  Give me a few minutes, and I should have it merged. :smile:

**Comment by lakme22** [comment]
Thanks a lot :+1:

**Comment by justinclift** [comment]
No worries at all. :smile:

**Comment by justinclift** [comment]
Done.  Manually added it with https://github.com/nickdesaulniers/What-Open-Source-Means-To-Me/commit/c2e7725d188869e64dc57788a4a1632becc9a56d. :smile:

**Comment by lakme22** [comment]
Thanks. Learned a lot :smile:

**Comment by nickdesaulniers** [comment]
Thanks @lakme22 for the contribution.  Thanks @justinclift for the fixup; git in general is not happy with our append to a file approach.

**Comment by justinclift** [comment]
Yeah, if we were to redo this repo from scratch again it'd likely be better to take some other approach.

Then again, if we had some kind of standard separator between the entries (eg `---`) that might work too.  I'm suspecting the problem git has, is that our diffs generally contain the last line of the previous entry, making them inflexible.  If the diffs instead didn't contain that, then the ordering would probably be much easier.

That's just a suspicion though.  It wouldn't be super hard to test (eg play around in a temp repo), but I've not gotten around to it. :wink:

**Pull request closed without merging**